include/mbedtls/debug.h

```diff
@@ -36,6 +36,28 @@
 #include "ecp.h"
 #endif
 
+/* TODO: Discuss how to test static functions!
+ * We need to have some static functions available
+ * in the test suites, and one way to deal with this
+ * would be to remove the static qualification by
+ * using a STATIC macro defined e.g. through
+ *
+ * #if defined(TEST_BUILD)
+ *  #define STATIC
+ *  #else
+ * #define STATIC static
+ * #endif
+ *
+ * We can't do this with TEST_BUILD == DEBUG_C
+ * at the moment, though, because there's a dependency
+ * of test_suite_mps on !DEBUG_C, as the printing
+ * routines in DEBUG_C need a valid SSL context, which
+ * the MPS test suite does not (and should not) provide.
```

Comment:
Note: to avoid `test_suite_mps` depending on `!DEBUG_C` we could make the debug functions exit early if passed `NULL` for the SSL context. (This is independent of the question of static functions, but I think this negative dependency is an annoyance.)

Reply:
Yes, I opened https://github.com/ARMmbed/mbedtls/pull/1818 for that.